### ISSUE TITLE
Suggestion: Display error message if query is bad

### ISSUE THREAD

**Issue by quantumgardener** [dsebastien/obsidian-dataview-serializer#18]
It took me a while to work out why some queries weren't updating. There was a small error somewhere but as a single line comment it was difficult to pick up.

I assume you can detect if there is an error because SeralizedQuery isn't displayed. If so, could you perhaps add at least a simple error message to ask the person to check the query? I'm sure it would help those who are less patient than I am 😉

**Comment by dsebastien** [comment]
Indeed, errors are only displayed in the console (at best), which is not very useful. We could add notifications when invalid queries are detected
